src/Api/Auth/Models/Request/Accounts/RotateUserKeyRequestModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Api.AdminConsole.Models.Request.Organizations;
+using Bit.Api.Tools.Models.Request;
+using Bit.Api.Vault.Models.Request;
+
+namespace Bit.Api.Auth.Models.Request.Accounts;
+
+public class RotateUserKeyRequestModel
+{
+    [Required][StringLength(300)] public string MasterPasswordHash { get; set; }
+    [Required] public string Key { get; set; }
```

Comment:
nit: shouldn't this be `UserKey`? Or is it implied 🤔

Reply:
I decided to not create a new endpoint and request model. This was a good suggestion, but making this change to an existing request model requires compatibility changes which might be out of scope right now.